With LuaJIT 2.1.0b3 `luaL_reg` have become `luaL_Reg`.
Switch it to fit the new LuaJIT API.

--- a/src/lib/elua/elua.c	2017-06-30 15:23:17.282654583 +0800
+++ b/src/lib/elua/elua.c	2017-06-30 15:24:23.929928673 +0800
@@ -355,7 +355,7 @@
    return 1;
 };
 
-const luaL_reg gettextlib[] =
+const luaL_Reg gettextlib[] =
 {
    { "bind_textdomain", _elua_gettext_bind_textdomain },
    { "get_message_language", _elua_get_message_language },
@@ -442,7 +442,7 @@
    return 1;
 }
 
-const luaL_reg _elua_cutillib[] =
+const luaL_Reg _elua_cutillib[] =
 {
    { "init_module", _elua_module_init },
    { "popenv"     , _elua_io_popen    },
--- a/src/lib/elua/io.c	2017-06-30 15:23:28.362763566 +0800
+++ b/src/lib/elua/io.c	2017-06-30 15:24:38.440046304 +0800
@@ -313,7 +313,7 @@
    return 1;
 }
 
-static const luaL_reg elua_popenlib[] =
+static const luaL_Reg elua_popenlib[] =
 {
    { "close"     , elua_close     },
    { "flush"     , elua_flush     },
